www/templates/refund.html

```diff
@@ -0,0 +1,35 @@
+<ion-view>
+  <ion-content>
+    <div class="tabulator" sf-resize  ng-controller="KeyPadCtrl">
+      <div class="section-1">
+        <div class="calc-component">
+          <div class="list">
+            <label class="item item-input item-select">
+              <div class="input-label">
+                Select Category
+              </div>
+              <select>
+                <option>None</option>
+                <option selected>Groceries</option>
+                <option>Website</option>
+              </select>
+            </label>
+          </div>
+        </div>
+
+        <div class="calc-component">
+          <div class="number-display">
+            <span class="number">{{amount | currency}}</span>
+          </div>
+        </div>
+      </div>
+
+      <div class="section-2">
+        <ng-include src="'templates/keypad.html'"></ng-include>
+        <div class="main-action button button-full button-positive">
+          Refund
+        </div>
+      </div>
+    </div>
+  </ion-content>
+</ion-view>
```

Comment:
There is a huge amount of duplication here with the charge page. Let's discuss how to better reuse this code.